### PR TITLE
Destroy resources documentation

### PR DESCRIPTION
**Pull request by ethanch9**
This request updates the ReadMe with instructions for the user of the module to destroy the untracked resources created by the sagemaker notebook. I have also added a missing argument to the module template.

### CODE REVIEW

**Review by konradbcredera — CHANGES_REQUESTED** [review]

**Review by konradbcredera — COMMENTED** [review]
reviewed

**Review by konradbcredera — CHANGES_REQUESTED** [review]
Review done

File: README.md [hunk 1]
```diff
@@ -82,4 +83,32 @@ No resources.
...
+
+```bash
+ - name: "Destroy Untracked Resources"
+          run: |
+            aws sagemaker delete-model --model-name < demo-regression-model >
+            aws sagemaker delete-endpoint-config --endpoint-config-name < demo-regression-model-config >
+            aws sagemaker delete-endpoint --endpoint-name < demo-regression-model >    
+```
+
+But before this you will need to add your AWS credentials to the environment if you have not do already:
```
[the remark this answers is not in the document]
Comment (ethanch9) [reply README.md:105]: I've just checked, you need the AWS credentials to use the aws command

[the remark this answers is not in the document]
Comment (konradbcredera) [reply README.md:103]: Ok. That's fine. Could we please delete the lines below, as the syntax would vary depending on the CI/CD tool?
```
- name: "Configure AWS credentials"
        uses: aws-actions/configure-aws-credentials@v1
        with:
```

Could we also please fix the indentation in both code blocks?

**Review by konradbcredera — APPROVED** [review]

**Pull request merged**